Tools/Gulp/gulpfile.js

```diff
@@ -77,6 +78,7 @@ gulp.task("buildNoWorker", ["shaders"], function () {
     )
     .pipe(concat(config.build.minNoWorkerFilename))
     .pipe(cleants())
+    .pipe(addModuleExports("BABYLON"))
```

Comment:
A few reasons - first, it's kinda scalable.
Second, I couldn't find the right place for this kind of file. Is it a TypeScript file? JavaScript? It doesn't belong to the framework.
It's just a small piece of code, I thought a small plugin will be the best. No one will wonder where it us coming from.